src/main/groovy/liquibase/harness/CompatibilityHarnessSuite.groovy

```diff
@@ -1,12 +1,12 @@
 package liquibase.harness
 
 
-import liquibase.harness.compatibility.basic.BasicCompatibilityTest
-import liquibase.harness.compatibility.foundational.FoundationalCompatibilityTest
+import liquibase.harness.compatibility.basic.CompatibilityTest
+import liquibase.harness.compatibility.foundational.FoundationalTest
 import org.junit.runner.RunWith
 import org.junit.runners.Suite
 
 @RunWith(Suite.class)
-@Suite.SuiteClasses([BasicCompatibilityTest, FoundationalCompatibilityTest])
+@Suite.SuiteClasses([CompatibilityTest, FoundationalTest])
 abstract class CompatibilityHarnessSuite {
```

Comment:
This suite name itself should renamed I think @AlexanderSashchenko, @KushnirykOleh -- Either include both Compatibility & Foundational in the name or remove both.